microsoft/featuremanagement/_featuremanager.py

```diff
@@ -41,6 +36,20 @@ def __init__(self, feature_flags, **kwargs):
                 raise ValueError("Custom filter must be a subclass of FeatureFilter")
             self._filters[filter.name] = filter
 
+    def _get_feature_flag(self, feature_flag_name):
+        feature_management = self._configuration.get(FEATURE_MANAGEMENT_KEY)
+        if not feature_management:
+            feature_management = self._configuration
```

Comment:
We were considering not having this fallback for the new schema- since no one has it yet we could force it to adhere to the schema definition fully.

Meaning C# will only work with Microsoft Feature Management schema if it's fully `"feature_management" { "feature_flags": { } }`. 

Not sure if we want to follow suit here.

Reply:
I'm personally not a huge fan of removing it as it's so easy to support... I would rather have it work than not.

I can bring it up with Jimmy and the rest to the team.

Reply:
I would start without it. Yes it is easy to support, but more options can sometimes be more harmful than helpful. More decisions for users of what's the "right thing to do", more things to think about when maintaining.

Since Python has the option to go without root fallback I would avoid it.

Reply:
@rossgrambo wanted to add, in the context of Python, this is not 'new schema' since it's the only one Python will support.

Reply:
Updated to return `None` if the schema isn't correct when getting the feature flag.